### PR TITLE
Add download files section 

### PR DESCRIPTION
**Pull request by allyhawkins**
Here I am starting to add in the text for the files download section to address #6. I included an overview of what information is included in each download and then broke it down into sections based on the four files that we are including - unfiltered counts matrix, filtered counts matrix, qc report, and then metadata. 

While writing this, I thought it would be helpful to include information on how to actually read in the RDS files and access the different pieces of the SCE (such as the additional metadata that we've included). I went ahead and started to add a FAQ for how to use these files in R here to address this. 

Additionally there were a few questions that came up while I was doing this that I would appreciate some feedback on: 
1. Right now we are including the `library_metadata.json` file for every library in the download, based on what's in the [google doc](https://docs.google.com/document/d/1hdYOXWWPOldnyvHDn_wjLw72g09BTs5xqOAsf6bRUzs/edit), but is this what we want to be doing? I'm not sure what added information this is giving to the user beyond the metadata available in `libraries_metadata.csv` and the metadata in the SCE object, so I don't know if this file needs to be user facing. 

2. I wasn't quite sure the level of detail to include here about the QC report. I kept it fairly simple to only describe what type of file it was and the general information that was present, but do we want a more in depth explanation of each of the plots here? or somewhere else in the docs? 

Also, there are a few points where I included links to other places in the docs, but I don't think I can actually include the links until we have RTD setup, so I put in placeholders for now. Let me know if this isn't the case and I can fix this.

### CODE REVIEW

**Comment by jashapiro** [comment]
> 1. Right now we are including the `library_metadata.json` file for every library in the download, based on what's in the [google doc](https://docs.google.com/document/d/1hdYOXWWPOldnyvHDn_wjLw72g09BTs5xqOAsf6bRUzs/edit), but is this what we want to be doing? I'm not sure what added information this is giving to the user beyond the metadata available in `libraries_metadata.csv` and the metadata in the SCE object, so I don't know if this file needs to be user facing.

I am not sure what the full content of libraries_metadata.csv will be. I assume that will have all of the sample info as well as the particular libraries, in which case I would agree that the metadata.json files may be redundant. But I think that there are fields in that file that might not be fully represented in the csv file or the metadata of the individual files. I would not want that data to only be in the metadata of the SCE objects or QC report, as that is a bit heavy to parse if someone is only interested in looking at some individual fields. So I would lean toward keeping them, as they are quite easy to open and read on a case by case basis.


> 2\. I wasn't quite sure the level of detail to include here about the QC report. I kept it fairly simple to only describe what type of file it was and the general information that was present, but do we want a more in depth explanation of each of the plots here? or somewhere else in the docs?

I hope we don't need much about it. The goal is for it to be pretty self-explanatory. If people have to go to the docs to figure out what we wrote in the QC report, I would be worried.

File: docs/download_files.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+| participant_id  | Original participant id, if there are multiple samples from the same participant                                                                          |
+| submitter       | Submitter name/id                                              |
+| age             | Age at time sample was obtained                                |
+| sex             | Sex of patient sample is obtained from                      |
+| diagnosis       | Tumor type                                                     |
+| subdiagnosis    | Subcategory of diagnosis or mutation status (if applicable)   |
+| tissue_location | Where tumor sample was located                                 |
+| disease_timing  | What stage of disease was the sample obtained? At diagnosis or recurrence? |
+
+Additional metadata may also be included that will be specific to the disease type. Examples of this include treatment or outcome.
```
Comment (jashapiro) [inline docs/download_files.md:58]: I think we want to be clear that this is as much project-specific as disease type. Not sure I love the wording.
```suggestion
Additional metadata may also be included, specific to the disease type and experimental design of the project. Examples of this include treatment or outcome. 
```

File: docs/download_files.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+| age             | Age at time sample was obtained                                |
+| sex             | Sex of patient sample is obtained from                      |
+| diagnosis       | Tumor type                                                     |
+| subdiagnosis    | Subcategory of diagnosis or mutation status (if applicable)   |
+| tissue_location | Where tumor sample was located                                 |
+| disease_timing  | What stage of disease was the sample obtained? At diagnosis or recurrence? |
+
+Additional metadata may also be included that will be specific to the disease type. Examples of this include treatment or outcome. 
+
+For each library, there will also be a `<scpca_library_id>_metadata.json`, containing processing associated metadata.
```
Comment (jashapiro) [inline docs/download_files.md:60]:
```suggestion
For each library, is also a `<scpca_library_id>_metadata.json` file that contains processing-associated metadata.
```

File: docs/faq.md [hunk 3]
```diff
@@ -18,4 +18,44 @@ We also compared the mean gene expression reported for each gene by both methods
...
\ No newline at end of file
+[You _et al._ (2021)](https://doi.org/10.1101/2021.06.17.448895) and [Tian _et al._ (2019)](https://doi.org/10.1038/s41592-019-0425-8) have also noted that results from different pre-processing workflows for single-cell RNA-sequencing analysis tend to result in compatible results downstream.
+
+#### How do I use the provided RDS files in R? 
+
+We are providing the gene expression data to you as a [`SingleCellExperiment`](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html) in an RDS file.
+
+_Note: You will need to install and load the [`SingleCellExperiment` package](https://bioconductor.org/packages/3.13/bioc/html/SingleCellExperiment.html) from Bioconductor to work with the provided files._
+
+To read in the RDS files you can use the following options.
```
Comment (jashapiro) [inline docs/faq.md:29]:
```suggestion
To read in the RDS files you can use the following commands. 
```

File: docs/faq.md [hunk 4]
```diff
@@ -18,4 +18,44 @@ We also compared the mean gene expression reported for each gene by both methods
...
+To read in the RDS files you can use the following options. 
+In base R, you can use the `readRDS` command. 
+```
+scpca_sample <- readRDS("SCPCL000001_filtered.rds")
+```
+
+Or in the `readr` package, you can use the `read_rds()` function. 
+```
+scpca_sample <- readr::read_rds("SCPCL000001_filtered.rds")
+```
```
Comment (jashapiro) [inline docs/faq.md:38]: I think we can just use `readRDS()` here. I don't think there is any advantage to `read_rds()` other than matching the tidyverse.

File: docs/faq.md [hunk 5]
```diff
@@ -18,4 +18,44 @@ We also compared the mean gene expression reported for each gene by both methods
...
+``` 
+
+The `SingleCellExperiment` holds the gene expression data as a matrix in the assays slot and any associated metadata. 
+
+To access the counts data directly you can do: 
+```
+counts(scpca_sample)
+```
+
+We have included metrics for each cell (associated with every column) and each gene (associated with each row).
```
Comment (jashapiro) [inline docs/faq.md:47]: Here I think we want to be clear about what those metrics are: we should probably reference `scuttle::addPerCellQCmetrics` and `miQC` for the `prob_compromised` column

**Review by jashapiro — COMMENTED** [review]
This looks like a great start, and I think most of the content is there. 

I left a lot of comments, largely focused on style. One theme: present tense everywhere, because by the time people read this the files will exist. I also tried to clarify a few things here and there, but I definitely don't consider anything I wrote as "correct". 

The RDS file faq section I would probably split into a separate PR. I think it needs more detail, but I think this PR is going to require enough time that I would rather separate that discussion. That or leave it skeletal here and we come back to it as a separate issue.

File: docs/download_files.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+## Unfiltered Counts Matrix 
+
+The unfiltered counts matrix, `<scpca_library_id>_unfiltered.rds`, is delivered as a RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
+Within the `SingleCellExperiment` is the counts matrix, where the rows correspond to genes or features and the columns correspond to cell barcodes. 
+Here, all potential cell barcodes that are identified after running [Alevin-fry](processing_information.html/#alignment-and-quantification-using-alevin-fry) are included in the counts matrix. 
+Summary statistics for each cell and gene can be found in the `colData` and `rowData` of the `SingleCellExperiment` object.
+The `SingleCellExperiment` object also contains metadata about that particular library, including the versions of Salmon and Alevin-fry used for pre-processing, information about the index used for transcriptome alignment, and parameters used for Alevin-fry. 
+See also [Using the provided RDS files in R.](link to FAQ)
+
+## Filtered Counts Matrix
```
[the remark this answers is not in the document]
Comment (allyhawkins) [reply docs/download_files.md:31]: I like the idea of putting these under one heading of `Gene expression data`. In terms of describing the colData and rowData, I wasn't sure how much detail we wanted to include here and am thinking that increasing the level of detail might belong in a section where we discuss addition of QC and metadata. I think #14 starts to address this, but we might want to add information about adding per cell and per feature QC in the processing section, rather than here?

File: docs/download_files.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+| column_id       | contents                                                       |
+|-----------------|----------------------------------------------------------------|
+| scpca_sample_id | Sample ID in the form `SCPCS000000`                            |
+| scpca_library_id | Library ID in the form `SCPCL000000`                          |
+| seq_unit        | `cell` for single-cell samples or `nucleus` for single-nucleus samples |
+| technology      | 10X kit used to process library                                |
+| filtered_cell_count | Number of cells after filtering with `emptyDrops`          |
+| submitter_id    | Original sample identifier from submitter                      |
+| participant_id  | Original participant id, if there are multiple samples from the same participant                                                                          |
```
Comment (kurtwheeler) [inline docs/download_files.md:57]: I don't think I've seen this one before. Where is it coming from?

File: docs/download_files.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
+# Downloadable files
+
+The ScPCA portal download packages include gene expression data, a QC report, and associated metadata for each processed sample.
```
Comment (jashapiro) [inline docs/download_files.md:3]:
```suggestion
The ScPCA Portal download packages include gene expression data, a QC report, and associated metadata for each processed sample.
```

File: docs/download_files.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+These files are delivered as a zip file.
+The folder structure within the zip file is determined by selecting to download all samples associated with a project or selecting individual samples.
```
Comment (jashapiro) [inline docs/download_files.md:5]: Trying to reword this to flow a bit better.
```suggestion
The folder structure within the zip file is determined by whether individual samples or all samples associated with a project are selected for download.  
```

File: docs/download_files.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+
+## Download folder structure for project downloads: 
+_add in illustration of file structure for project_ 
+
+## Download folder structure for individual sample downloads: 
+_add in illustration of file structure for sample downloads_ 
+
+Sample folders (indicated by the `SCPCS` prefix) contain the files for all libraries (`SCPCL` prefix) derived from that biological sample. 
+Most samples only have one library that has been sequenced. 
+See {ref}`the difference between samples and libraries <faq:what is the difference between a sample and a library?>` for more information.
```
Comment (jashapiro) [inline docs/download_files.md:15]: I felt like the link text here was a bit confusing.
```suggestion
See the {ref}`FAQ section about samples and libraries <faq:what is the difference between a sample and a library?>` for more information.
```

File: docs/download_files.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+
+The files associated with each library are:
+- An unfiltered counts file: `SCPCL000000_unfiltered.rds`, 
+- A filtered counts file: `SCPCL000000_filtered.rds`, 
+- A quality control report: `SCPCL000000_qc.html`, 
+- A metadata file: `SCPCL000000_metadata.json`.
```
Comment (jashapiro) [inline docs/download_files.md:21]: Just to be sure the markdown is parsed correctly.
```suggestion
- A metadata file: `SCPCL000000_metadata.json`. 

```

File: docs/download_files.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+_add in illustration of file structure for project_ 
+
+## Download folder structure for individual sample downloads: 
+_add in illustration of file structure for sample downloads_ 
+
+Sample folders (indicated by the `SCPCS` prefix) contain the files for all libraries (`SCPCL` prefix) derived from that biological sample. 
+Most samples only have one library that has been sequenced. 
+See {ref}`the difference between samples and libraries <faq:what is the difference between a sample and a library?>` for more information.
+
+The files associated with each library are:
```
Comment (jashapiro) [inline docs/download_files.md:17]: Not sure about the wording here, but trying to be a bit more explicit here.
```suggestion
The files associated with each library are (shown for a library with ID `SCPCL000000`):
```

File: docs/download_files.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+- An unfiltered counts file: `SCPCL000000_unfiltered.rds`, 
+- A filtered counts file: `SCPCL000000_filtered.rds`, 
+- A quality control report: `SCPCL000000_qc.html`, 
+- A metadata file: `SCPCL000000_metadata.json`. 
+Every download also includes a `libraries_metadata.csv` file containing metadata associated with each library included in the download.
```
Comment (jashapiro) [inline docs/download_files.md:22]: Trying to clarify here that this is for multiple libraries, not one per library. 

```suggestion
Every download also includes a single `libraries_metadata.csv` file containing metadata for all libraries included in the download.
```

File: docs/download_files.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+
+## Gene expression data
+
+Gene expression data is available in two forms - as an unfiltered counts file or filtered counts file.
```
Comment (jashapiro) [inline docs/download_files.md:26]:
```suggestion
Gene expression data is provided in two forms - as an unfiltered counts file and a filtered counts file.
```

File: docs/download_files.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+
+The unfiltered counts file, `SCPCL000000_unfiltered.rds`, is delivered as a RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
```
Comment (jashapiro) [inline docs/download_files.md:28]: Simplifying a bit.
```suggestion
The unfiltered counts file, `SCPCL000000_unfiltered.rds`, is an RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
```

File: docs/download_files.md [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+Here, all potential cell barcodes that are identified after running {ref}`Alevin-fry <processing_information:alignment and quantification using alevin-fry>` are included in the counts matrix. 
+Summary statistics for each cell and gene can be found in the `colData` and `rowData` slots of the `SingleCellExperiment` object, respectively.
+The `SingleCellExperiment` object also contains metadata about that particular library, including the versions of Salmon and Alevin-fry used for pre-processing, information about the index used for transcriptome alignment, and parameters used for Alevin-fry. 
+See also {ref}`Using the provided RDS files in R <faq:how do i use the provided RDS files in r?>`.
+
+The filtered counts file is also delivered as a RDS file containing a `SingleCellExperiment` object.
+Following filtering using {ref}`emptyDrops <processing_information:filtering cells>`, the filtered counts matrix and updated summary statistics for each cell and gene are output to the `SCPCL000000_filtered.rds` file.
+As a result, this file only contains cell barcodes that are considered true cells.
```
Comment (jashapiro) [inline docs/download_files.md:37]: I'm trying to keep this paragraph more parallel with the previous one.
```suggestion
The filtered counts file, `SCPCL000000_filtered.rds` is also an RDS file containing a `SingleCellExperiment` object with the same structure as above.
The cells in this file are those that remain after filtering using {ref}`emptyDrops <processing_information:filtering cells>`.
As a result, this file only contains cell barcodes that are likely to correspond to true cells.
```

File: docs/download_files.md [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+
+See {ref}`gene expression file contents <gene_expression_file_contents:gene expression file contents>` for more information.
```
Comment (jashapiro) [inline docs/download_files.md:39]: Adding a bit more detail.
```suggestion
See {ref}`gene expression file contents <gene_expression_file_contents:gene expression file contents>` for more information about the contents of the `SingleCellExperiment` objects and the included statistics and metadata.
```

File: docs/download_files.md [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+
+## Gene expression data
+
+Gene expression data is available in two forms - as an unfiltered counts file or filtered counts file.
+
+The unfiltered counts file, `SCPCL000000_unfiltered.rds`, is delivered as a RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
+Within the `SingleCellExperiment` object is the counts matrix, where the rows correspond to genes or features and the columns correspond to cell barcodes. 
+Here, all potential cell barcodes that are identified after running {ref}`Alevin-fry <processing_information:alignment and quantification using alevin-fry>` are included in the counts matrix. 
+Summary statistics for each cell and gene can be found in the `colData` and `rowData` slots of the `SingleCellExperiment` object, respectively.
+The `SingleCellExperiment` object also contains metadata about that particular library, including the versions of Salmon and Alevin-fry used for pre-processing, information about the index used for transcriptome alignment, and parameters used for Alevin-fry.
```
Comment (jashapiro) [inline docs/download_files.md:32]: I am not sure how much detail we want to put into this, since we will cover it in a separate section. I would definitely cut down the second sentence, as I am suggesting here. I'm also removing the references to `colData` and `rowData`, but I feel less strongly about that. 

```suggestion
The object also includes summary statistics for each cell barcode and gene, as well as metadata about that particular library, such as the reference index and software versions used for mapping and quantification. 
```

File: docs/download_files.md [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+
+Gene expression data is available in two forms - as an unfiltered counts file or filtered counts file.
+
+The unfiltered counts file, `SCPCL000000_unfiltered.rds`, is delivered as a RDS file containing a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html).
+Within the `SingleCellExperiment` object is the counts matrix, where the rows correspond to genes or features and the columns correspond to cell barcodes. 
+Here, all potential cell barcodes that are identified after running {ref}`Alevin-fry <processing_information:alignment and quantification using alevin-fry>` are included in the counts matrix. 
+Summary statistics for each cell and gene can be found in the `colData` and `rowData` slots of the `SingleCellExperiment` object, respectively.
+The `SingleCellExperiment` object also contains metadata about that particular library, including the versions of Salmon and Alevin-fry used for pre-processing, information about the index used for transcriptome alignment, and parameters used for Alevin-fry. 
+See also {ref}`Using the provided RDS files in R <faq:how do i use the provided RDS files in r?>`.
```
Comment (jashapiro) [inline docs/download_files.md:33]: I think I would move this sentence to after the "see gene expression file contents", or combine the two.

File: docs/download_files.md [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+| technology      | 10X kit used to process library                                |
+| filtered_cell_count | Number of cells after filtering with `emptyDrops`          |
+| submitter_id    | Original sample identifier from submitter                      |
+| participant_id  | Original participant id, if there are multiple samples from the same participant                                                                          |
+| submitter       | Submitter name/id                                              |
+| age             | Age at time sample was obtained                                |
+| sex             | Sex of patient sample is obtained from                      |
+| diagnosis       | Tumor type                                                     |
+| subdiagnosis    | Subcategory of diagnosis or mutation status (if applicable)   |
+| tissue_location | Where tumor sample was located                                 |
```
Comment (jashapiro) [inline docs/download_files.md:63]: Maybe? I don't think this is really ambigous, but it read strangely to me. 
```suggestion
| tissue_location | Where in the body the tumor sample was located                                 |
```

File: docs/download_files.md [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+| scpca_sample_id | Sample ID in the form `SCPCS000000`                            |
+| scpca_library_id | Library ID in the form `SCPCL000000`                          |
+| seq_unit        | `cell` for single-cell samples or `nucleus` for single-nucleus samples |
+| technology      | 10X kit used to process library                                |
+| filtered_cell_count | Number of cells after filtering with `emptyDrops`          |
+| submitter_id    | Original sample identifier from submitter                      |
+| participant_id  | Original participant id, if there are multiple samples from the same participant                                                                          |
+| submitter       | Submitter name/id                                              |
+| age             | Age at time sample was obtained                                |
+| sex             | Sex of patient sample is obtained from                      |
```
Comment (jashapiro) [inline docs/download_files.md:60]: Consistent tense, mostly.
```suggestion
| sex             | Sex of patient that the sample was obtained from                      |
```

File: docs/download_files.md [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+| diagnosis       | Tumor type                                                     |
+| subdiagnosis    | Subcategory of diagnosis or mutation status (if applicable)   |
+| tissue_location | Where tumor sample was located                                 |
+| disease_timing  | What stage of disease was the sample obtained? At diagnosis or recurrence? |
+
+Additional metadata may also be included, specific to the disease type and experimental design of the project. Examples of this include treatment or outcome.
```
Comment (jashapiro) [inline docs/download_files.md:66]: Just breaking at sentence.
```suggestion
Additional metadata may also be included, specific to the disease type and experimental design of the project.
Examples of this include treatment or outcome. 
```

File: docs/faq.md [hunk 23]
```diff
@@ -18,4 +18,18 @@ We also compared the mean gene expression reported for each gene by both methods
...
+
+#### How do I use the provided RDS files in R? 
+
+We are providing the gene expression data to you as a [`SingleCellExperiment` object](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html) in an RDS file.
+
+_Note: You will need to install and load the [`SingleCellExperiment` package](https://bioconductor.org/packages/3.13/bioc/html/SingleCellExperiment.html) from Bioconductor to work with the provided files._
+
+To read in the RDS files you can use the following commands. 
+In base R, you can use the `readRDS` command. 
+```
```
Comment (jashapiro) [inline docs/faq.md:31]: Do we want to include the library loading here? It may not be necessary, but I thought we might include it as a point of reference.  

````suggestion

```
library(SingleCellExperiment)
````

File: docs/faq.md [hunk 24]
```diff
@@ -18,4 +18,18 @@ We also compared the mean gene expression reported for each gene by both methods
...
+scpca_sample <- readRDS("SCPCL000001_filtered.rds")
```
Comment (jashapiro) [inline docs/faq.md:32]: For consistency with the rest of the document.
```suggestion
scpca_sample <- readRDS("SCPCL000000_filtered.rds")
```

File: docs/faq.md [hunk 25]
```diff
@@ -18,4 +18,18 @@ We also compared the mean gene expression reported for each gene by both methods
 
 Recent reports from others support our findings. 
 [He _et al._ (2021)](https://doi.org/10.1101/2021.06.29.450377)) demonstrated that Alevin-fry can process single-cell and single-nuclei data more quickly and efficiently then other available methods, while also decreasing the false positive rate of gene detection that is commonly seen in methods that utilize transcriptome alignment.
-[You _et al._ (2021)](https://doi.org/10.1101/2021.06.17.448895) and [Tian _et al._ (2019)](https://doi.org/10.1038/s41592-019-0425-8) have also noted that results from different pre-processing workflows for single-cell RNA-sequencing analysis tend to result in compatible results downstream.
\ No newline at end of file
+[You _et al._ (2021)](https://doi.org/10.1101/2021.06.17.448895) and [Tian _et al._ (2019)](https://doi.org/10.1038/s41592-019-0425-8) have also noted that results from different pre-processing workflows for single-cell RNA-sequencing analysis tend to result in compatible results downstream.
+
+#### How do I use the provided RDS files in R?
```
Comment (jashapiro) [inline docs/faq.md:23]: A note here but also applies elswhere... Which level of heading do we want here? Is there a reason not to stick with just H2 (`##`)?

File: docs/faq.md [hunk 26]
```diff
@@ -18,4 +18,18 @@ We also compared the mean gene expression reported for each gene by both methods
...
+
+_Note: You will need to install and load the [`SingleCellExperiment` package](https://bioconductor.org/packages/3.13/bioc/html/SingleCellExperiment.html) from Bioconductor to work with the provided files._
+
+To read in the RDS files you can use the following commands. 
+In base R, you can use the `readRDS` command. 
+```
+scpca_sample <- readRDS("SCPCL000001_filtered.rds")
+```
+
+### What is the difference between a sample and a library?
```
Comment (jashapiro) [inline docs/faq.md:35]: Noting inconsistent header depth here.

File: docs/faq.md (hunk 25 above)
Comment (allyhawkins) [reply docs/faq.md:23]: Just seeing this after requesting your re-review. Let me put all the headers to be H2 and then it will be ready for re-review.

**Review by jashapiro — APPROVED** [review]
LGTM!
